src/Features/ERDDAP/Superlatives/index.tsx

```diff
@@ -0,0 +1,136 @@
+/**
+ * Superlatives, or what platforms experiencing the most extreme conditions right now
+ */
+import * as React from "react"
+import { Link } from "react-router-dom"
+import { Card, CardBody, CardHeader, Col, Row } from "reactstrap"
+
+import { useUnitSystem } from "Features/Units"
+import { converter } from "Features/Units/Converter"
+import { UnitSystem } from "Features/Units/types"
+import { paths } from "Shared/constants"
+import { round } from "Shared/math"
+import { urlPartReplacer } from "Shared/urlParams"
+
+import { UsePlatforms } from "../hooks"
+import { PlatformFeature, PlatformTimeSeries } from "../types"
+import { conditions } from "../Platform/Observations/CurrentConditions/conditions"
+
+const waveHeight = new Set(conditions.waveHeight)
+const windSpeed = new Set(conditions.windSpeed)
+
+/**
+ * Superlatives, or what platforms experiencing the most extreme conditions right now
+ */
+export const Superlatives: React.FunctionComponent = () => {
+  const unitSystem = useUnitSystem()
+
+  const lastHour = new Date()
+  lastHour.setUTCHours(lastHour.getUTCHours() - 1)
+
+  return (
+    <UsePlatforms>
+      {({ platforms }) => <ShowSuperlatives laterThan={lastHour} {...{ platforms, unitSystem }} />}
```

Comment:
Codacy found an issue: ['ShowSuperlatives' was used before it was defined.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6227039)